content/docs/command-reference/exp/show.md

```diff
@@ -127,6 +129,14 @@ metric or param.
 
 - `--md` - prints the command's output in Markdown table format.
 
+- `--html` - generates an interactive parallel coordinates plot from the table.
+
+- `-o <folder>, --out <folder>` - destination folder of the `html` plot. By
+  default, `dvc_plots`.
```

Comment:
> Question about default value in https://github.com/iterative/dvc/pull/6933#issuecomment-996068883

Reply:
If kept here, it should state it has no effect without `--html`.

Reply:
Could we have a L4 title that says `PCP options` and lists these two together?

Reply:
Not sure about adding a title, it might look like the other options don't have effect on the plot (but some do)

Reply:
> should state it has no effect without (`--pcp`)

I'll update this. Resolving.